cli_tools/common/utils/storage/scratch_bucket_creator.go

```diff
@@ -26,8 +26,9 @@ import (
 )
 
 const (
-	defaultRegion       = "US"
-	defaultStorageClass = "MULTI_REGIONAL"
+	defaultRegion        = "US"
```

Comment:
since what is default is determined programatically now, this should be renamed to defaultMultiregion

Reply:
They are still default. We are still using defaultRegion and defaultStorageClass when we can't determine region by any input.
Other fallback region/zone value is infured from input, so I don't call them "default".
Please let me know whether that sounds correct.